protobuf/types/process.proto

```diff
@@ -91,7 +92,10 @@ message Process {
         oneof value {
           // Input defined as reference.
           Reference ref = 2 [
-            (gogoproto.moretags) = 'hash:"name:2" validate:"required"'
+            (gogoproto.moretags) = 'hash:"name:2" validate:"required_without=Constant"'
```

Comment:
What is the case to put `require_without=Constant` here?

Reply:
to keep the check that ref is required. I cannot put just required because if constant is given then ref will be nil so the required will fail. In that case, just say that it's required only when there is no constant